jgclark.DailyJournal/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # What's changed in ☀️ Daily Journal Plugin?
 
+## [0.9.0] - 2021-11-25
+### Changes
```

Comment:
I think the standard term 'Added' is more appropriate than 'Changed' here :-)

Reply:
yes you are right :)